vpr/src/analytical_place/global_placer.h

```diff
@@ -0,0 +1,135 @@
+/**
+ * @file
+ * @author  Alex Singer
+ * @date    October 2024
+ * @brief   The declarations of the Global Placer base class which is used to
+ *          define the functionality of all global placers in the AP flow.
+ *
+ * Global Placers create a Partial Placement given onlythe netlist and the
```

Comment:
Global placers create -> A global placer creates
onlythe -> only the
uses analytical -> uses analytical (i.e. efficient numerical minimization of an objective function of a placement)